src/udev/cdrom_id/cdrom_id.c

```diff
@@ -26,49 +27,85 @@
 #include "udev-util.h"
 
 /* device info */
+static unsigned cd_rw_nonremovable;
+static unsigned cd_rw_removable;
 static unsigned cd_cd_rom;
 static unsigned cd_cd_r;
 static unsigned cd_cd_rw;
+static unsigned cd_ddcd_rom;
+static unsigned cd_ddcd_r;
+static unsigned cd_ddcd_rw;
 static unsigned cd_dvd_rom;
 static unsigned cd_dvd_r;
+static unsigned cd_dvd_r_ddr;
+static unsigned cd_dvd_r_dl;
+static unsigned cd_dvd_r_dl_seq;
+static unsigned cd_dvd_r_dl_jr;
 static unsigned cd_dvd_rw;
+static unsigned cd_dvd_rw_ro;
+static unsigned cd_dvd_rw_seq;
+static unsigned cd_dvd_rw_dl;
 static unsigned cd_dvd_ram;
 static unsigned cd_dvd_plus_r;
 static unsigned cd_dvd_plus_rw;
 static unsigned cd_dvd_plus_r_dl;
 static unsigned cd_dvd_plus_rw_dl;
 static unsigned cd_bd;
 static unsigned cd_bd_r;
+static unsigned cd_bd_r_srm;
+static unsigned cd_bd_r_rrm;
 static unsigned cd_bd_re;
 static unsigned cd_hddvd;
 static unsigned cd_hddvd_r;
+static unsigned cd_hddvd_r_dl;
 static unsigned cd_hddvd_rw;
+static unsigned cd_hddvd_rw_dl;
 static unsigned cd_mo;
+static unsigned cd_mo_se;
+static unsigned cd_mo_wo;
+static unsigned cd_mo_as;
 static unsigned cd_mrw;
 static unsigned cd_mrw_w;
 
 /* media info */
 static unsigned cd_media;
+static unsigned cd_media_rw_nonremovable;
+static unsigned cd_media_rw_removable;
 static unsigned cd_media_cd_rom;
 static unsigned cd_media_cd_r;
 static unsigned cd_media_cd_rw;
+static unsigned cd_media_ddcd_rom;
+static unsigned cd_media_ddcd_r;
+static unsigned cd_media_ddcd_rw;
 static unsigned cd_media_dvd_rom;
 static unsigned cd_media_dvd_r;
+static unsigned cd_media_dvd_r_ddr; /* download disc recording - dvd for css managed recording */
+static unsigned cd_media_dvd_r_dl;
+static unsigned cd_media_dvd_r_dl_seq; /* sequential recording */
+static unsigned cd_media_dvd_r_dl_jr; /* jump recording */
 static unsigned cd_media_dvd_rw;
 static unsigned cd_media_dvd_rw_ro; /* restricted overwrite mode */
 static unsigned cd_media_dvd_rw_seq; /* sequential mode */
+static unsigned cd_media_dvd_rw_dl;
 static unsigned cd_media_dvd_ram;
 static unsigned cd_media_dvd_plus_r;
 static unsigned cd_media_dvd_plus_rw;
 static unsigned cd_media_dvd_plus_r_dl;
 static unsigned cd_media_dvd_plus_rw_dl;
 static unsigned cd_media_bd;
 static unsigned cd_media_bd_r;
+static unsigned cd_media_bd_r_srm; /* sequential recording mode */
+static unsigned cd_media_bd_r_rrm; /* random recording mode */
 static unsigned cd_media_bd_re;
 static unsigned cd_media_hddvd;
 static unsigned cd_media_hddvd_r;
+static unsigned cd_media_hddvd_r_dl;
 static unsigned cd_media_hddvd_rw;
+static unsigned cd_media_hddvd_rw_dl;
 static unsigned cd_media_mo;
+static unsigned cd_media_mo_se; /* sector erase */
+static unsigned cd_media_mo_wo; /* write once */
+static unsigned cd_media_mo_as; /* advance storage */
 static unsigned cd_media_mrw;
 static unsigned cd_media_mrw_w;
```

Comment:
hmm, so thee are all booleans? given you touch it maybe make them all C99 bools now. It's how we have been doing for new code in the rest of the codebase, this is just some old leftover. But when you touch this, please update it.

That said, maybe a flags field might actually be better here, given how many those are now

Reply:
My change is using same coding style and same approach as existing code.

So this looks like a technical debt in your project which is not relevant to the issue which I'm trying to fix.

I do not have a time nor capacity to remove this technical debt in your project.

Reply:
OK, let's do that coding style change later.